File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n = 0;
    //take input between 1 and 8
    do
    {
        n = get_int("input numbers between 1 and 8: ");
    }
    while (n > 8 || n < 1);

    //for input 8 -> loop 7
    for (int i = 1; i <= n; i++)
    {
        for (int j = 1; j <= n; j++)
        {
            //for n-i print " "
            if (j <= n - i)
            {
                printf(" ");
            }
            //for i print "#"
            else
            {
                printf("#");
            }
        }
        printf("\n");
    }

}

File: sentimental-mario-more/mario.py
from cs50 import get_int

# TODO


# Take input between 1~8
while True:
    h = get_int("Height: ")
    if h < 9 and h > 0:
        break

for i in range(0, h):
    for j in range(0, h-i-1):
        print(" ", end="")
    print("#" * (i+1), end="")
    print("  ", end="")
    print("#" * (i+1), end="")
    print("")
